addons/mail/static/src/core/web/mail_composer_template_selector.scss

```diff
@@ -0,0 +1,11 @@
+.mail-composer-template-dropdown {
+    max-width: 250px !important;
+
+    @include media-breakpoint-up(sm) {
+        max-width: 400px !important;
```

Comment:
LGTM :+1:

I'm not entirely sure why the `mw-100` class was added. I noticed there's a CSS variable `--popover-max-width`, and it's unclear why that wasn't reused or set directly. :thinking: Using the variable would have made it easier to adjust the popover's max width without resorting to the `!important` keyword.

As for the stable version, I agree that we should avoid modifying it, since changes there would affect everyone. So, I'm fine with overriding the max width in this way for now.

Regarding the breakpoints and the values you chose: they seem to work well across all screen resolutions. :smiley: